superset/assets/src/visualizations/core/models/ChartMetadata.js

```diff
@@ -2,14 +2,22 @@ export default class ChartMetadata {
   constructor({
     name,
     credits = [],
-    description,
-    thumbnail,
+    description = '',
     show = true,
+    canBeAnnotation = [],
```

Comment:
`canBeAnnotationTypes` / `supportedAnnotationTypes` might be marginally clearer, but I'm okay with `canBeAnnotation` / `supportedAnnotations`, too. 

if going with `canBeAnnotation` / `supportedAnnotations`, maybe we should also make `canBeAnnotation` plural for consistency? => `canBeAnnotations`?

Reply:
I like `canBeAnnotationTypes` / `supportedAnnotationTypes`. Was overthinking about singular/plural thing. I think your suggestion makes sense.